scripts/dev/templates/versionhook/Dockerfile.builder

```diff
@@ -1,4 +1,5 @@
-FROM golang:1.18.5 as builder
+ARG builder_image
+FROM ${builder_image} as builder
 
 ENV GO111MODULE=on
```

Comment:
Not related to your changes but since you have this "under your axe" - could you please delete those unnecessary Go env. variables?